packages/fx-core/src/common/azure-hosting/azureFunctionHosting.ts

```diff
@@ -7,6 +7,7 @@ import { TokenProvider, Void } from "@microsoft/teamsfx-api";
 import { azureWebSiteDeploy } from "./utils";
 import { AzureOperations } from "./azureOps";
 import { getResourceGroupNameFromResourceId, getSiteNameFromResourceId } from "../tools";
+import { Messages } from "./messages";
```

Comment:
There is a code scanning warning of "Unused import Messages."

Reply:
Fixed.